service/raft/group0_state_machine.cc

```diff
@@ -91,7 +91,8 @@ future<> group0_state_machine::apply(std::vector<raft::command_cref> command) {
         merger(group0_state_machine& sm_, utils::UUID id, semaphore_units<> mux) : last_group0_state_id(id)
```

Comment:
commit message:
"i`max_mutation_size`" (what's the "i" for)
"Fixe" -> "Fixed"

I would start the sentence "A batch of raft commands has the size at most..." on a new paragraph, so it's clearer that it talks about a different thing (the code inside `group0_state_machine`) than the previous part (which was talking about the code in `raft_sys_table_storage`).

Reply:
Fixed.